### PR TITLE
Move existing code to Desktop.Analyzers 

### PR DESCRIPTION
**Pull request by genlu**
- Move helpers to `Analyzer.Utility`
- Move existing files to `Desktop.Analyzers`: CA2229, CA2235, CA2237

@srivatsn @mavasani @lgolding @michaelcfanning

### CODE REVIEW

File: PortFxCop/src/Analyzer.Utilities/CommonAccessibilityUtilities.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 using Microsoft.CodeAnalysis;
 
-namespace Microsoft.AnalyzerPowerPack.Utilities
+namespace Analyzer.Utilities
```
Comment (lgolding) [inline PortFxCop/src/Analyzer.Utilities/CommonAccessibilityUtilities.cs]: Is that really the root namespace for this project? I guess I didn't notice that. No "Microsoft" at the front? @michaelcfanning, @srivatsn for opinion.

In any case would not change in this PR.

File: PortFxCop/src/Analyzer.Utilities/Extensions/INamedTypeSymbolExtensions.cs [hunk 2]
```diff
@@ -30,6 +30,12 @@ public static bool DoesOverrideEquals(this INamedTypeSymbol symbol)
             return symbol.GetMembers(WellKnownMemberNames.ObjectEquals).OfType<IMethodSymbol>().Where(m => IsEqualsOverride(m)).Any();
         }
 
+        public static bool DoesOverrideGetHashCode(this INamedTypeSymbol symbol)
```
Comment (lgolding) [inline PortFxCop/src/Analyzer.Utilities/Extensions/INamedTypeSymbolExtensions.cs]: `OverridesGetHashCode`

**Comment by lgolding** [comment]
:+1:

**Comment by genlu** [comment]
I have create issue https://github.com/dotnet/roslyn-analyzers/issues/355 to track the naming problems for `analyzer.Utilities` project.

**Pull request merged**